charts/tidb-operator/templates/admission/admission-webhook-rbac.yaml

```diff
@@ -39,6 +39,10 @@ rules:
   - apiGroups: ["apps.pingcap.com"]
     resources: ["statefulsets"]
     verbs: ["*"]
+  - apiGroups: ["admissionregistration.k8s.io"]
+    resources: ["validatingwebhookconfigurations"]
+    verbs: ["delete"]
```

Comment:
I'm not familiar with helm. It seems that we need extra permission which is not part of our admission webhook because of `pre-delete-job`, do we have an alternative solution for this helm problem?
cc @aylei @weekface @onlymellb

Reply:
The pre-delete-job should have its own service-account which had `validatingwebhookconfigurations` permission and the admission webhook server won't have this permission.
To get rid of requesting `validatingwebhookconfigurations` permission, we should make helm delete `validatingwebhookconfigurations` first when we execute `helm del --purge`

Reply:
better we can clean by running `helm del --purge`. add a service-account for `pre-delete-job`?

Reply:
updated